server/sonar-db-dao/src/main/java/org/sonar/db/rule/DeprecatedRuleKeyDto.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.db.rule;
+
+import java.util.Objects;
+import javax.annotation.CheckForNull;
+
+/**
+ * Map the table "deprecated_rule_keys"
+ */
+public class DeprecatedRuleKeyDto {
+  /**
+   *  Uuid of the deprecated key
+   */
+  private String uuid;
+  /**
+   * the id of the current rule for this deprecated key
+   */
+  private Integer ruleId;
+  /**
+   * repository key that was deprecated
+   */
+  private String oldRepositoryKey;
+  /**
+   * rule key that was deprecated
+   */
+  private String oldRuleKey;
+  /**
+   * creation date of the record
```

Comment:
I don't understand "record".

"row" instead? (as this object maps to a table rows)